app.js

```diff
@@ -1,11 +1,82 @@
 /* Imports */
 
 /* Get DOM Elements */
+const cityArticle = document.getElementById('city-article');
+
+const nameInput = document.getElementById('name-input');
+const nameDisplay = document.getElementById('name-display');
+const climateSelect = document.getElementById('climate-select');
+const climateDisplay = document.getElementById('climate-image');
+const architectureSelect = document.getElementById('architecture-select');
+const architectureDisplay = document.getElementById('architecture-image');
+const attractionInput = document.getElementById('attraction-input');
+const addButton = document.getElementById('add-button');
+const attractionList = document.getElementById('attraction-list');
 
 /* State */
+let city = {
+    name: 'Cool Town',
+    climate: 'desert',
+    architecture: 'modern',
+    attractions: [],
+};
+
+/* Helper Functions */
+function pageLoad() {
+    displayControls();
+    displayCity();
+    displayAttractions();
+}
 
 /* Events */
+nameInput.addEventListener('input', () => {
+    city.name = nameInput.value;
+    pageLoad();
+});
```

Comment:
there's a fun little bug in this section — if you click the input and start typing, the text from the initial city name stays there and goes after what you're typing. there are two fixes: rather than having ```city.name = nameInput.value```, you could give the ```<input>``` an attribute of ```placeholder="Cool Town"``` which would display a grey text that disappears when user focuses on input field.
Alternately, you could probably do something like ```nameInput.textContent = ''``` in the first line of your eventListener